ArchiSteamFarm/Bot.cs

```diff
@@ -635,6 +649,80 @@ internal sealed class Bot : IDisposable {
 			return await ArchiWebHandler.GetTradeHoldDurationForTrade(tradeID).ConfigureAwait(false);
 		}
 
+		internal async Task<OrderedDictionary> GetUsedKeys() {
+			if (!File.Exists(KeysToRedeemUsedFilePath)) {
+				return new OrderedDictionary();
+			}
+
+			try {
+				OrderedDictionary usedKeys = new OrderedDictionary();
+
+				using (StreamReader reader = new StreamReader(KeysToRedeemUsedFilePath)) {
+					string line;
+
+					while ((line = await reader.ReadLineAsync().ConfigureAwait(false)) != null) {
+						if (line.Length == 0) {
+							continue;
+						}
+
+						string[] parsedArgs = line.Split(DefaultBackgroundKeysRedeemerSeparator, StringSplitOptions.RemoveEmptyEntries);
+						if (parsedArgs.Length < 4) {
+							ArchiLogger.LogGenericWarning(string.Format(Strings.ErrorIsInvalid, line));
+							continue;
+						}
+
+						string status = parsedArgs[1].Substring(1, parsedArgs[1].Length - 2);
```

Comment:
What does it do and why? You should just include name alone, no need for status. If you want to include status then include it as one string next to the name, just how ASF would if you supplied this `.unused` keys file for another bot BGR.